RELEASES.md

```diff
@@ -1,3 +1,200 @@
+Version 1.91.0 (2025-10-30)
+==========================
+
+<a id="1.91.0-Language"></a>
+
+Language
+--------
+
+- [Lower pattern bindings in the order they're written and base drop order on primary bindings' order](https://github.com/rust-lang/rust/pull/143764)
+- [Stabilize declaration of c-style variadic functions for sysv64, win64, efiapi, and aapcs ABIs](https://github.com/rust-lang/rust/pull/144066).
+  This brings these ABIs in line with the C ABI: variadic functions can be declared in extern blocks but not defined.
+- [Always require coroutine captures to be drop-live](https://github.com/rust-lang/rust/pull/144156)
+- [Add lint against dangling pointers from local variables](https://github.com/rust-lang/rust/pull/144322)
+- [Upgrade semicolon_in_expressions_from_macros from warn to deny](https://github.com/rust-lang/rust/pull/144369)
+- [Stabilize loongarch32 inline asm](https://github.com/rust-lang/rust/pull/144402)
+- [Add warn-by-default `integer_to_ptr_transmutes` lint against integer to pointer transmutes](https://github.com/rust-lang/rust/pull/144531)
+- [Stabilize `sse4a` and `tbm` target features](https://github.com/rust-lang/rust/pull/144542)
+- [Add `target_env = "macabi"` and `target_env = "sim"` cfgs](https://github.com/rust-lang/rust/pull/139451) as replacements for the `target_abi` cfgs with the same values.
+
+<a id="1.91.0-Compiler"></a>
+
+Compiler
+--------
+
+- [Error on invalid `#[link]` attributes](https://github.com/rust-lang/rust/pull/143193)
```

Comment:
The `#[link]` attribute is in this section while `#[should_panic]` is in the compatibility notes. Feels to me like this should be moved to compatibility notes